libbeat/scripts/Makefile

```diff
@@ -118,7 +118,7 @@ clean:: ## @build  Cleans up all files generated by the build steps
 	# Clean index pattern
 	rm -f $(PWD)/_meta/kibana/index-pattern/${BEAT_NAME}.json
 	# Remove temp templates
-	-rm ${BEAT_NAME}.template*.json
```

Comment:
I should have been more accurate here. It is still possible to generate the template through `make index-template` so removing can be still needed. But it (should) have no effect on the build as it is not used.